File: Lib/Telerik/Examples/QuickStart/GridView/ManipulateData/Expressions/CustomExpressionContext.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.Data.Expressions;

namespace Telerik.Examples.WinControls.GridView.ManipulateData.Expressions
{
    public class CustomExpressionContext : ExpressionContext
    {
        /// <summary>
        /// Custom function, which returns Pi.
        /// </summary>
        /// <returns></returns>
        public double Pi()
        {
            return Math.PI;
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/GridView/ManipulateData/Expressions/CustomExpressionContext.vb
Imports System.Text
Imports Telerik.Data.Expressions

Namespace Telerik.Examples.WinControls.GridView.ManipulateData.Expressions
	Public Class CustomExpressionContext
		Inherits ExpressionContext
		''' <summary>
		''' Custom function, which returns Pi.
		''' </summary>
		''' <returns></returns>
		Public Function Pi() As Double
			Return Math.PI
		End Function
	End Class
End Namespace
